frontend/cypress.config.js

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3001',
+    specPattern: "cypress/e2e/**/*.spec.js",
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+  },
+});
```

Comment:
```suggestion
});

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo para que ferramentas como o GitHub não exibam o ícone de `No newline at end of file`.

Isso é uma convenção e boa prática a ser seguida.

Reply:
O mesmo vale para outros arquivos.

Reply:
Sempre percebi que o vscode faz isso por automático mas nunca entendi o porquê. Como eu não entendia e uma linha em branco a mais no final do código não fazia sentido pra mim, sempre apaguei kk. Mas obrigado pela dica!